### PR TITLE
use platform prefix by default

### PR DESCRIPTION
**Pull request by kevinushey**
Closes https://github.com/rstudio/renv/issues/1211.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 
 # renv 0.18.0  (UNRELEASED)
 
+* **Breaking Change**: `renv` now includes platform prefix in library paths
```
Comment (hadley) [inline NEWS.md:4]:
```suggestion
* **Breaking Change**: `renv` now includes platform prefix in cache paths
```
?

**Review by hadley — APPROVED** [review]
Isn't this going to break the cache for every existing linux user? Is that ok?

File: NEWS.md (hunk 1 above)
Comment (kevinushey) [reply NEWS.md:4]: In fact, the change effects both:

```
> renv:::renv_paths_library()
[1] "/home/kevin/.cache/R/renv/library/renv-3022e2f8/linux-ubuntu-jammy/R-4.1/aarch64-unknown-linux-gnu"
> renv:::renv_paths_cache()
[1] "/tmp/Rtmpoiba2M/renv-tempfile-2ac82649a830d/cache/v5/linux-ubuntu-jammy/R-4.1/aarch64-unknown-linux-gnu"
```

**Comment by kevinushey** [comment]
> Isn't this going to break the cache for every existing linux user? Is that ok?

Yeah, you're right... do you have an opinion on what we should do here? I suppose we could drop a symlink from the "old" cache location to the "new" cache location on startup, to ensure the previous cache location remains visible?

File: NEWS.md (hunk 1 above)
Comment (hadley) [reply NEWS.md:4]: Oh so it's not just the cache that will break, but every existing project will need to re-install all packages? I think that's a pretty strong argument for adding some help to automate the transition.

**Comment by hadley** [comment]
Yeah, that was my feeling too.

**Pull request closed without merging**